src/lib.rs

```diff
@@ -203,23 +203,58 @@ pub struct Metrics {
     pub strikeout_thickness: f32,
 }
 
-pub trait Rasterize {
-    /// Errors occurring in Rasterize methods.
-    type Err: ::std::error::Error + Send + Sync + 'static;
+/// Errors occuring when using the rasterizer.
+#[derive(Debug)]
+pub enum Error {
+    /// Couldn't find font matching description.
+    MissingFont(FontDesc),
```

Comment:
I'm not sure if "Missing" is the correct word here? Maybe something like `FontNotFound`, akin to `std::io::ErrorKind::NotFound`?